### PR TITLE
PWGCF: FemtoUniverse track-V0 task -- added rectangular CPR cut and removed CPR for mixed events

### PR DESCRIPTION
**Pull request by Eloviyo**
- CPR check: added rectangular cut for v0-v0 pairs
- added CPR check back for mixed events
- removed unused configurable

### CODE REVIEW

File: PWGCF/FemtoUniverse/Core/FemtoUniverseDetaDphiStar.h [hunk 1]
```diff
@@ -225,8 +225,6 @@ class FemtoUniverseDetaDphiStar
       for (int i = 0; i < 2; i++) {
         auto indexOfDaughterpart1 = (ChosenEventType == femtoUniverseContainer::EventType::mixed ? part1.globalIndex() : part1.index()) - 2 + i;
         auto indexOfDaughterpart2 = (ChosenEventType == femtoUniverseContainer::EventType::mixed ? part2.globalIndex() : part2.index()) - 2 + i;
-        // auto indexOfDaughterpart1 = part1.globalIndex() - 2 + i;
```
Comment (zchochul) [inline PWGCF/FemtoUniverse/Core/FemtoUniverseDetaDphiStar.h:228]: Remove unused code

File: PWGCF/FemtoUniverse/Core/FemtoUniverseDetaDphiStar.h [hunk 2]
```diff
@@ -239,7 +237,8 @@ class FemtoUniverseDetaDphiStar
           LOG(fatal) << "FemtoUniverseDetaDphiStar: passed arguments don't agree with FemtoUniverseDetaDphiStar's type of events! Please provide same or mixed.";
         }
 
-        if (pow(dphiAvg, 2) / pow(CutDeltaPhiStarMax, 2) + pow(deta, 2) / pow(CutDeltaEtaMax, 2) < 1.) {
+        // if (pow(dphiAvg, 2) / pow(CutDeltaPhiStarMax, 2) + pow(deta, 2) / pow(CutDeltaEtaMax, 2) < 1.) {
```
[the remark this answers is not in the document]
Comment (zchochul) [reply PWGCF/FemtoUniverse/Core/FemtoUniverseDetaDphiStar.h:240]: Have you asked if the changes okay with @alicja-pp @basiach and @mm1707?

File: PWGCF/FemtoUniverse/Core/FemtoUniverseDetaDphiStar.h (hunk 1 above)
Comment (zchochul) [reply PWGCF/FemtoUniverse/Core/FemtoUniverseDetaDphiStar.h:228]: This is just a comment for later PR, I can approve it without this change. Crucial thing for me is green light from @alicja-pp @basiach and @mm1707.

Comment (zchochul) [reply PWGCF/FemtoUniverse/Core/FemtoUniverseDetaDphiStar.h:228]: I see that @alicja-pp and @mm1707 agreed (@basiach is not available) so I am approving the changes.

**Review by zchochul — DISMISSED** [review]

**Review by victor-gonzalez — APPROVED** [review]

**Pull request merged**